<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheNewEntreprise</class>
 <widget class="QDialog" name="TheNewEntreprise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>881</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Nouvelle Entreprise</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitreLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>The Gestion</string>
   </property>
  </widget>
  <widget class="QFrame" name="InfoWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>241</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Mon Compte</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="SurnameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatutLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Statut</string>
    </property>
   </widget>
   <widget class="QLabel" name="MailLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QLabel" name="TelLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifInfoButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifMdpButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Mot De Passe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GestionAssoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion De l'Asso</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nomEntrepriseLabel">
      <property name="text">
       <string>NomEntreprise</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="linenomEntreprise"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="siretLabel">
      <property name="text">
       <string>Siret :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineSiret"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="correspondantLabel">
      <property name="text">
       <string>Correspondant :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineCorrespondant"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telLabel">
      <property name="text">
       <string>Tel :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineTel"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="mailLabel">
      <property name="text">
       <string>Mail :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LineMail"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonValid">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonQuit">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
